Export_GetTablesToBeExported() if table exist

bkbalci opened the pull request:
exception thrown when table is not exist on source database so i added table exist control to code.

adriancs2 commented:
I agree with the implementation of pre-checking existing table list before applying the custom list from the dictionary or list.

This project is currently not using LinQ, as I still wish to support for .NET Framework 2.0 at the moment.

About the escape string sequence of eliminating the following characters:
```
'? and ’
```
Why there is a need to escape this? It won't break the SQL syntax.
Or do you misunderstood the usage of this function `"EscapeStringSequence(string data)"`

bkbalci commented:
Hi,

I forgot about .net framework 2.0 sorry.

As for the special characters i added, they break the syntax when i trying
to import backup so thats why i added them to list.

Thanks for reply.

18 May 2019 Cmt 05:26 tarihinde Adrian Voo <notifications@github.com> şunu
yazdı:

> I agree with the implementation of pre-checking existing table list before
> applying the custom list from the dictionary or list.
>
> This project is currently not using LinQ, as I still wish to support for
> .NET Framework 2.0 at the moment.
>
> About the escape string sequence of eliminating the following characters:
>
> '? and ’
>
>
> Why there is a need to escape this? It won't break the SQL syntax.
> Or do you misunderstood the usage of this function "EscapeStringSequence(string
> data)"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MySqlBackupNET/MySqlBackup.Net/pull/42?email_source=notifications&email_token=ALHVJ7YHYRBG2MDQIRBVLZTPV5SNZA5CNFSM4HNKFNE2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODVWF6IY#issuecomment-493641507>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALHVJ7Z3WL3PD7H7HF6RIGDPV5SNZANCNFSM4HNKFNEQ>
> .
>

adriancs2 commented:
Hi, thanks for the bug report and fixed code for the 1st problem: exception thrown when table is not exist on source database.
I have fixed the problem.

but however, about your second proposal of fix: escape string sequence of 
```
? and ’
```
I'm currently not able to reproduce the error. You can try again with MySqlDump, it seems that MySqlDump also not escaping the string sequence for these 2 characters too.

If you strongly believe that these 2 characters should be escaped, please provide evidence, perhaps a sample project that demonstrate the proof.

Thanks.
